bloomfilter/tests/bloomfilter-tests.hs

```diff
@@ -103,10 +106,10 @@ proxyBlocked = Proxy
 
 prop_elem :: forall bloom a. (BloomFilter bloom, Hashable a)
           => Proxy bloom -> Proxy a
-          -> a -> [a] -> FPR -> Property
-prop_elem proxy _ x xs (FPR q) =
+          -> B.Salt -> a -> [a] -> FPR -> Property
+prop_elem proxy _ salt x xs (FPR q) =
     let bf :: bloom a
-        bf = fromList (policyForFPR proxy q) (x:xs)
+        bf = fromList salt (policyForFPR proxy q) (x:xs)
```

Comment:
I don't know quite why it irks me, having the salt first and size second, but it does :grin:.